### ISSUE TITLE
 PhoBERT (phobert-large) model suddenly outputs all "O" labels after some training epochs

### ISSUE THREAD

**Issue by quangdaist01** [VinAIResearch/PhoNER_COVID19#3]
I'm trying to replicate the results on the datasets, but my Colab Pro cannot fit into the batch_size of 32 (as in the paper). So I decreased the batch_size down to 4.
I just train PhoBERT model on the dataset. The results of phobert-base model is quite impressive. 
However, phobert-large model starts of training at a lower eval_f1 score than phobert-base
Here is my training details:
```
!python /content/transformers/examples/legacy/token-classification/run_ner.py \
--task_type NER \
--overwrite_output_dir True \
--data_dir '/content/' \
--labels '/content/labels.txt' \
--model_name_or_path 'vinai/phobert-large' \
--output_dir '/content/drive/MyDrive/Môn học/NLP materials/Do an/test-ner/phobert-large' \
--learning_rate 5e-5 \
--max_seq_length  128 \
--num_train_epochs 30 \
--per_device_train_batch_size 4 \
--per_device_eval_batch_size 4 \
--save_total_limit=100 \
--seed 1 \
--do_train \
--do_eval \
--do_predict \
--logging_strategy epoch \
--evaluation_strategy epoch \
--save_strategy epoch \
--lr_scheduler_type 'constant' \
--metric_for_best_model 'eval_f1' \
--logging_dir '/content/drive/MyDrive/Môn học/NLP materials/Do an/test-ner/phobert-large'  \
--load_best_model_at_end True \
```

![image](https://user-images.githubusercontent.com/40832101/143148942-3196b5e4-977c-4a01-9979-2947fb2691f9.png)

Before using run_ner.py provided by huggingface team, I manually encoded labels to the splitted (sub-word) tokens.
The results were the same. I thought it was because of some mistakes while encoding but it doesn't seem to be the case.

Is the small batch_size causing this or something else?

**Comment by quangdaist01** [comment]
Here is the result I got when I trained phobert-large using my own script (not run_ner.py)
![image](https://user-images.githubusercontent.com/40832101/143149981-066cd746-d26e-4789-b64d-af8a71271af0.png)

**Comment by datquocnguyen** [comment]
It's likely because you're using a large learning rate. 
Please try different (smaller) learning rates, e.g. 5e-6.

**Comment by quangdaist01** [comment]
Thank you for your response. I'll give it a try. Have a nice day!
